extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -60,25 +59,21 @@ let BoundPropertyDef;
 let BoundElementDef;
 
 /**
- * @param {!Node|!../../../src/service/ampdoc-impl.AmpDoc} nodeOrAmpDoc
- * @return {!Bind}
- */
-export function installBindForTesting(nodeOrAmpDoc) {
-  return fromClassForDoc(nodeOrAmpDoc, 'bind', Bind);
-}
-
-/**
  * Bind is the service that handles the Bind lifecycle, from identifying
  * bindings in the document to scope mutations to reevaluating expressions
  * during a digest.
  */
 export class Bind {
   /**
    * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
+   * @param {Object=} opt_mode
    */
-  constructor(ampdoc) {
+  constructor(ampdoc, opt_mode) {
+    /** @private {!Object} */
+    this.mode_ = opt_mode || getMode(ampdoc.win);
```

Comment:
Cannot store `getMode`'s return value. You have to always grab an individual property from the object. Don't worry, it's optimized to be `true` or `false` in prod builds.